src/main/java/duke/DialogBox.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import javafx.scene.shape.Circle;
+import javafx.scene.text.Font;
+
+public class DialogBox extends HBox {
+
+    private Label text;
+    private ImageView displayPicture;
+
+    /**
+     * Initializes a DialogBox with Label as text content & ImageView as Profile Image.
+     * DialogBox always force enable word wrap for Label and prevents truncations.
+     *
+     * @param l  String textContent
+     * @param iv Profile Image for DialogBox
+     */
+    public DialogBox(Label l, ImageView iv) {
```

Comment:
I feel that it might be better if you use variable names that are more descriptive especially for such a long method.

Reply:
Justified. 👍